src/SwcCompiler.ts

```diff
@@ -77,7 +88,7 @@ export class SwcCompiler implements Compiler {
     const group = this.compiledFiles.group(filename);
 
     if (!group) {
-      throw new Error(await this.missingDestination(filename));
+      throw new MissingDestinationError(await this.missingDestination(filename));
```

Comment:
We had already considered changing `compile` so that it returned the one file immediately. After using it to write the tests, I found it clunky that `compile` wouldn't crash when compiling the file I asked for, and only crashed in `fileGroup`. Unless we preserve some more state (files that failed to build, reasons, etc), it makes it much harder to produce good error messages, and we're left with unknown state. Only the logs show the reason.

Reply:
:+1: makes sense to me to make this API actually useful.